src/SprykerEco/Zed/Computop/ComputopConfig.php

```diff
@@ -8,6 +8,7 @@
 namespace SprykerEco\Zed\Computop;
 
 use Spryker\Zed\Kernel\AbstractBundleConfig;
+use SprykerEco\Shared\Computop\ComputopConfig as ConputopSharedConfig;
```

Comment:
`ConputopSharedConfig` => `ComputopSharedConfig`.

Reply:
++

Reply:
I don't see this changes.

Reply:
The typo is already fixed.